include/openPMD/Series.hpp

```diff
@@ -20,22 +20,25 @@
  */
 #pragma once
 
-#include "openPMD/config.hpp"
-#include "openPMD/auxiliary/Deprecated.hpp"
-#include "openPMD/backend/Attributable.hpp"
-#include "openPMD/backend/Container.hpp"
 #include "openPMD/IO/AbstractIOHandler.hpp"
 #include "openPMD/IO/Access.hpp"
 #include "openPMD/IO/Format.hpp"
 #include "openPMD/Iteration.hpp"
 #include "openPMD/IterationEncoding.hpp"
+#include "openPMD/Streaming.hpp"
+#include "openPMD/auxiliary/Deprecated.hpp"
+#include "openPMD/auxiliary/Option.hpp"
+#include "openPMD/auxiliary/Variant.hpp"
+#include "openPMD/backend/Attributable.hpp"
+#include "openPMD/backend/Container.hpp"
+#include "openPMD/config.hpp"
 #include "openPMD/version.hpp"
 
 #if openPMD_HAVE_MPI
 #   include <mpi.h>
 #endif
 
-#include <string>
+#include <map>
```

Comment:
`std::string` is still used in this file:
```suggestion
#include <map>
#include <string>
```